src/controller/servicemgr/executor/containerexecutor/containerexecutor.go

```diff
@@ -63,7 +64,7 @@ func GetInstance() *ContainerExecutor {
 }
 
 // Execute executes container service application
-func (c ContainerExecutor) Execute(s executor.ServiceExecutionInfo) error {
+func (c *ContainerExecutor) Execute(s executor.ServiceExecutionInfo) error {
 	c.ServiceExecutionInfo = s
```

Comment:
Do we need to have `executor.ServiceExecutionInfo` in the [`type ContainerExecutor struct`](https://github.com/lf-edge/edge-home-orchestration-go/blob/9db48c416d504c668a5fad22a0b80f67ab07a506/src/controller/servicemgr/executor/containerexecutor/containerexecutor.go#L45)?
I think we can execute a container with `s` and don't need to save it to `c.ServiceExecutionInfo`.
How do you think?

Reply:
I think this information may be useful in the future. Right now I have good reason to delete it.

Reply:
> @t25kim I think this information may be useful in the future. Right now I have good reason to delete it. Thank you for question

@tdrozdovsky Please recommend to post up the suggestion from @t25kim, so that we can follow up the regarding consideration in the future, even though we adopt this PR as it is. What do you think? Once we see the posted issue regarding this, I will merge this PR.